app/services/build_runner_service.rb

```diff
@@ -28,6 +28,10 @@ def add_build_runner(build_runner: nil)
       return task
     end
 
+    def remove_build_runner(build_runner:)
+      build_runners.delete(build_runner)
```

Comment:
This might not be thread-safe 😨

Reply:
Just checked, Arrays aren't thread-safe, but many interpreters are broken-ish (depending on who you ask 🙃) and don't do threading properly, so arrays accidentally become thread-safe. We shouldn't rely on that though and we should use proper locking for this.